pkgs/by-name/je/jellyfin-ffmpeg/package.nix

```diff
@@ -8,14 +8,55 @@ let
   version = "7.0.2-5";
 in
 
-(ffmpeg_7-full.override {
+(ffmpeg_7-headless.override {
   inherit version; # Important! This sets the ABI.
   source = fetchFromGitHub {
     owner = "jellyfin";
     repo = "jellyfin-ffmpeg";
     rev = "v${version}";
     hash = "sha256-cqyXQNx65eLEumOoSCucNpAqShMhiPqzsKc/GjKKQOA=";
   };
+
+  # https://github.com/NixOS/nixpkgs/pull/354936#issuecomment-2466628369
+  # --enable-gmp # not supported by our ffmpeg drv
+  withGnutls = true;
+  withChromaprint = true;
+  withOpencl = true;
+  withDrm = true;
+  withXml2 = true;
+  withAss = true;
+  withFreetype = true;
+  withFribidi = true;
+  withFontconfig = true;
+  withHarfbuzz = true;
+  withBluray = true;
+  withMp3lame = true;
+  withOpus = true;
+  withTheora = true;
+  withVorbis = true;
+  withOpenmpt = true;
+  withDav1d = true;
+  withSvtav1 = true;
+  withWebp = true;
+  withVpx = true;
+  withX264 = true;
+  withX265 = true;
+  withZvbi = true;
+  withZimg = true;
+
+  withShaderc = true;
+  withPlacebo = true;
+  withVulkan = true;
+  withVaapi = true;
+  withAmf = true;
+  withVpl = true;
+  withMfx = false;
+  withNvcodec = true;
+  # --enable-cuda = true; # unfree
```

Comment:
```
// non-free, provided by the closed source NVCC compiler from Cuda SDK
--enable-cuda-nvcc       enable Nvidia CUDA compiler [no]

// free, provided by Clang/LLVM compiler
--disable-cuda-llvm      disable CUDA compilation using clang [autodetect]
```
https://github.com/FFmpeg/FFmpeg/blob/7f51cf75c6efb6a358b4cdb683cfbd8370d56a8f/configure#L344-L345

Reply:
Indeed, I thought this was nvcc flag. Thanks!